client/core/core.go

```diff
@@ -2480,6 +2503,20 @@ func (c *Core) Register(form *RegisterForm) (*RegisterResult, error) {
 		return nil, codedError(connectionErr, err)
 	}
 
+	// Ensure this DEX supports this asset for registration fees, and get the
+	// required confirmations and fee amount.
+	dc.cfgMtx.RLock()
+	feeAsset, supported := dc.cfg.RegFees[regFeeAssetSymbol]
+	dc.cfgMtx.RUnlock()
+	if !supported || feeAsset == nil {
+		return nil, newError(assetSupportErr, "dex server does not accept registration fees in asset %q", regFeeAssetSymbol)
+	}
+	reqConfs := feeAsset.Confs
+	if reqConfs > 1000 { // basic sanity check on required confirms
```

Comment:
Could be less, and per asset.

Reply:
I might want to eliminate this new check.  It is asset-specific as you said, and it's kind out of scope for this PR, at least to come up with a per-asset limit.